### PR TITLE
docs: update and reorganize readme.md

### PR DESCRIPTION
**Pull request by bmartinez287**
## The Issue
https://github.com/vuejs/vue
https://github.com/withastro/astro
https://github.com/laravel/laravel
https://github.com/lando/lando
https://github.com/tailwindlabs/tailwindcss
https://github.com/vercel/next.js/

After looking at some popular open-source repos it seems like we could switch some items around. 

## How This PR Solves The Issue

This PR switches the order of certain items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,42 @@
...
 
-DDEV is an open source tool for running local web development environments for PHP, Python and Node.js, ready in minutes. Its powerful, flexible per-project environment configurations can be extended, version controlled, and shared. DDEV allows development teams to adopt a consistent Docker workflow without the complexities of bespoke configuration.
+DDEV is an open-source tool for running local web development environments for PHP, Python and Node.js, ready in minutes. Its powerful, flexible per-project environment configurations can be extended, version controlled, and shared. DDEV allows development teams to adopt a consistent Docker workflow without the complexities of bespoke configuration.
+
+## Wonderful Sponsors
+
+DDEV is an Apache License 2.0 open-source project with its ongoing development made possible entirely by the support of these awesome backers. If you'd like to join them, please consider sponsoring DDEV development.
```
Comment (rfay) [inline README.md:15]:
```suggestion
DDEV is an Apache License 2.0 open-source project with its ongoing development made possible entirely by the support of these awesome backers. If you'd like to join them, please consider [sponsoring DDEV development](https://github.com/sponsors/ddev).
```

File: README.md [hunk 2]
```diff
@@ -2,21 +2,42 @@
...
+  <source media="(prefers-color-scheme: dark)" srcset="https://ddev.com/resources/featured-sponsors-darkmode.svg">
+  <img alt="DDEV Sponsor logos with light and dark mode variants" src="https://ddev.com/resources/featured-sponsors.svg">
+</picture>
+
+## Documentation
+To check out live examples, docs, contributor live training, guides and more visit [ddev.com](https://ddev.com) and [ddev.readthedocs.io](https://ddev.readthedocs.io/en/latest/users/support)
+
+## Questions
+
+If you need help, our friendly community provides [great support](https://ddev.readthedocs.io/en/latest/users/support). The issue list of this repo is exclusively for bug reports and feature requests.
```
Comment (rfay) [inline README.md:27]:
```suggestion
If you need help, our friendly community provides [great support](https://ddev.readthedocs.io/en/latest/users/support). 
```

File: README.md [hunk 3]
```diff
@@ -2,21 +2,42 @@
...
+
+## Contributing
+
+See “How can I contribute to DDEV?” in the [FAQ](https://ddev.readthedocs.io/en/latest/users/usage/faq/), and the [Contributing](CONTRIBUTING.md) page.
+
+![Overview of GitHub contributions](https://repobeats.axiom.co/api/embed/941b040a17921e974655fc01d7735aa350a53603.svg "Repobeats analytics image")
 
 ## Get Started
```
Comment (rfay) [inline README.md:36]: Consider linking instead (or in addition) to ddev.com, which has a super friendly get started. https://ddev.com/get-started/

Comment (bmartinez287) [reply README.md:36]: I added a note after the other getting-started items.

**Review by rfay — CHANGES_REQUESTED** [review]
This is looking great and super concise, I like it.

https://github.com/ddev/ddev/blob/94080ee1b4f7f7b9b6ae518852dfd4f1153015c0/README.md

The one thing I'd ask is to move the "Wonderful Sponsors" below "Questions". I think we want people to get right to the meat and see the sponsors as they keep reading. But because it's so concise they get there just fine.

**Comment by bmartinez287** [comment]
Done.

**Comment by mattstein** [comment]
If you both like this by all means run with it, but I’m struggling to understand the problem and how reordering things solves it.

The reference repos order things differently, and IMO this buries what DDEV is and how you might start doing something with it—which seem like the most urgent things to convey to someone arriving at the project.

**Comment by rpkoller** [comment]
I can second matt's point. from the perspective of a person landing the first time on the project page the top tasks are what is ddev (the introductory paragraph), what are it's capabilities (highlighted features), and how do i get started (get started).  but in the current version you have: 

1. introductory paragraph
2. documentation 
3. questions
4.. wonderful sponsors
5. contributing
6. get started
7. highlighted features. 

For someone new to the project the point of interest starts at point 6. and i am not sure how more experienced ddev users are using the readme or if they use the readme at all since they use either ddev.readthedocs.com and/or ask on one of the socials. (by the way the socials aren't listed in the readme?)

**Comment by bmartinez287** [comment]
I can see that. But every project has stages, early on the get started on the readme was likely the main starting point. Now we have one on the docs, one on ddev.com and one here. 
I think we might be at the stage where emphasizing sponsors and highlighting them might make more sense.
So those looking to get started have multiple ways of acquiring that knowledge but to sustain the project and achieve the goal of acquiring more full time maintainers we need to highlight sponsors more prominently in my opinion.

**Comment by bmartinez287** [comment]
I can see that. But every project has stages, early on the get started on the readme was likely the main starting point. Now we have one on the docs, one on ddev.com and one here. 
I think we might be at the stage where emphasizing sponsors and highlighting them might make more sense.
So those looking to get started have multiple ways of acquiring that knowledge but to sustain the project and achieve the goal of acquiring more full time maintainers we need to highlight sponsors more prominently in my opinion.

**Comment by bmartinez287** [comment]
Data analytics on this would be great way of testing it. But I'm not sure how that can be achieved on a repo.

We could always try it and if it does not seem to work change it back later.

**Comment by rfay** [comment]
I think it's a good contribution, at least freshening things up a bit. Followups very much appreciated.

It's easy to fuss about a contribution like this, but followup PRs with specific points will be welcome.

**Pull request merged**